protos/sources/vega/events/v1/events.proto

```diff
@@ -1154,6 +1167,9 @@ enum BusEventType {
   // Event indicating a funding period has ended and resulted in funding payment transfers.
   BUS_EVENT_TYPE_FUNDING_PAYMENTS = 80;
 
+  // Event indicating the paid liquidity fees stats per market at end of epoch
```

Comment:
This description doesn't read right to me - the event indicates the paid fees statistics?

Reply:
well, they are the statistics for the paid liquidity fees per market per epoch. Please could you suggest a better way to say it? Happy to merge it immediately

Reply:
```suggestion
  // Event used to report the updated stats for the market at the end of the epoch






```